mlir/test/Dialect/Arith/expand-ops-scale.mlir

```diff
@@ -0,0 +1,159 @@
+// RUN: mlir-opt %s -arith-expand -split-input-file -verify-diagnostics | FileCheck %s
```

Comment:
Did you intent for this to go into this PR? Looks unrelated to me and should probably go to a separate PR.